bot/exts/filters/webhook_remover.py

```diff
@@ -9,12 +9,15 @@
 from bot.exts.moderation.modlog import ModLog
 from bot.utils.messages import format_user
 
-WEBHOOK_URL_RE = re.compile(r"((?:https?://)?discord(?:app)?\.com/api/webhooks/\d+/)\S+/?", re.IGNORECASE)
+WEBHOOK_URL_RE = re.compile(
+    r"((?:https?:\/\/)?(?:ptb\.|canary\.)?discord(?:app)?\.com\/api\/webhooks\/\d+\/)\S+\/?",
+    re.IGNORECASE
+)
 
 ALERT_MESSAGE_TEMPLATE = (
     "{user}, looks like you posted a Discord webhook URL. Therefore, your "
-    "message has been removed. Your webhook may have been **compromised** so "
-    "please re-create the webhook **immediately**. If you believe this was a "
+    "message has been removed, and your webhook has been deleted."
```

Comment:
I don't understand, I saw the screenshot of this message which looked like there was a space after this period.
```suggestion
    "message has been removed, and your webhook has been deleted. "
```